scripts/commit-and-pr.ps1

```
@@ -0,0 +1,75 @@
+    [string]$BaseBranch = "main",
+    [string]$PrTitle = "",
+    [string]$PrBody = ""
+)
+
+Set-StrictMode -Version Latest
+
+function Exec($args) {
+    Write-Host "> $args"
+    & cmd /c $args
+    $code = $LASTEXITCODE
+    if ($code -ne 0) { throw "Command failed with exit code $code: $args" }
```

PSScriptAnalyzer (1.25.0)

[error] Variable reference is not valid. ':' was not followed by a valid variable name character. Consider using ${} to delimit the name.

(InvalidVariableReferenceWithDrive)